apps/pxweb2/src/app/app.module.scss

```diff
@@ -77,3 +77,9 @@
   max-width: 100%;
   min-width: 100%;
 }
+
+.scrollable {
+  height: 100vh;
+  align-self: stretch;
+  overflow-y: auto;  
```

Comment:
There is a blank space at the end here, can you format the file?

Reply:
Blankspace removed

Reply:
Sorry, I wasn't clear here. I meant on line 84, there are 2 blank spaces after the ";"